File: labs/PHYS291_A/compiles/comp.c
// file     comp.c
// compile with   
//             gcc -o C_comp comp.c
// run by
//             ./C_comp
//
#include <stdio.h>
int main(void)
{
  int i;
  for(i=0;i<20;i++)
printf("%d  %d \n", i, i*i);
return 0;
}

File: labs/PHYS291_A/compiles/comp.f
C  file comp.f
C  SIMPLE Fortran example
C  compile by    gfortran -o fortran_comp comp.f
C  run by:       ./fortran_comp
C
      integer i
      do i=1,10
             write(*,'( I3, 3x, I5 )') i, i*i
      end do
      end


